liger_iris_pipeline/tests/test_image2.py

```diff
@@ -1,38 +1,58 @@
-import json
-import pytest
+# Imports
+import liger_iris_pipeline
+liger_iris_pipeline.monkeypatch_jwst_datamodels()
 import numpy as np
-import warnings
-import os
-from astropy.utils import data
+from liger_iris_pipeline.tests.test_utils import get_data_from_url
+import json
 from jwst.associations import load_asn
 
-import iris_pipeline
-
-iris_pipeline.monkeypatch_jwst_datamodels()
 
-from iris_pipeline.tests.test_utils import get_data_from_url
+######################
 
 def test_image2():
-    with open("iris_pipeline/tests/data/asn_subtract_bg_flat.json") as fp:
+
+    # Load the association file
+    # Contains a science and sky exposure to process
+    with open("liger_iris_pipeline/tests/data/asn_subtract_bg_flat.json") as fp:
         asn = load_asn(fp)
-    raw_science_filename = get_data_from_url("17903858")
-    raw_background_filename = get_data_from_url("17903855")
+
+    # Download the raw science frame
+    raw_science_filename = get_data_from_url("48191524")
```

Comment:
it would be nice to have the URL on figshare of these files if someone wants to inspect them

Reply:
I have a list of the URLs with basic info - what do you think about it to the readme?

Reply:
yes, good idea, also put a note here to look at the readme for info about the test files.

Reply:
I added them to the README in [fe7d0c9](https://github.com/oirlab/liger_iris_pipeline/pull/75/commits/fe7d0c9823f1b187770c4bcb1e575f543963ab9a). I think a note above each call to `get_data_from_url` is redundant for us for the time being. Eventually we can revisit this if needed.

Reply:
what about a note on top of each test file that uses get_data_from_url?

Reply:
Added the same note to each test script that uses `get_data_from_url` in [7b979ce](https://github.com/oirlab/liger_iris_pipeline/pull/75/commits/7b979ce69603445e7c060cc74aab1e25be71d081)